plenum/test/input_validation/node_status_db_validation/test_restore_last_sent_pp_seq_no.py

```diff
@@ -0,0 +1,68 @@
+from logging import getLogger
+
+import pytest
+
+from common.serializers.serialization import node_status_db_serializer
+from plenum.common.constants import LAST_SENT_PRE_PREPARE
+from plenum.test.input_validation.helper import OutputWarningHandler
+from plenum.test.primary_selection.test_primary_selector import FakeNode
+
+logger = getLogger()
+container = list()
+logger.addHandler(OutputWarningHandler(container))
+warning_msg_count = len(container)
+initial_3pc = None
+
+
+@pytest.fixture(scope="module")
+def node(tconf, tdir):
+    node = FakeNode(tdir, tconf)
+    global initial_3pc
+    initial_3pc = node.master_replica.last_ordered_3pc
+    return node
+
+
+def check_last_warning(expected_msg):
+    global warning_msg_count
+    warning_msg_count += 1
+    assert warning_msg_count == len(container)
```

Comment:
I would rather prefer error codes (instead of just true/false) instead of analyzing logs